CMakeLists.txt

```diff
@@ -64,11 +65,17 @@ set(CMAKE_C_FLAGS   "-g3 -Wall -Werror ${CMAKE_C_FLAGS}")
 set(CMAKE_CXX_FLAGS "-g3 -Wall -Werror ${CMAKE_CXX_FLAGS}")
 include_directories(include)
 
+enable_testing()
+
 add_subdirectory(src)
 add_subdirectory(java)
 
 # Enable Testing
-enable_testing()
```

Comment:
I moved it to line 68 (I can't remember why).

2016-06-20 11:25 GMT+02:00 Martin Quinson <notifications@github.com>:

> In CMakeLists.txt
> <https://github.com/simgrid/remote-simgrid/pull/6#discussion_r67659244>:
>
> >  add_subdirectory(src)
> >  add_subdirectory(java)
> >
> >  # Enable Testing
> > -enable_testing()
>
> Why do you desactivate the tests?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/simgrid/remote-simgrid/pull/6/files/22ebce8766ac65a67ab24fe39be6559afe893556#r67659244>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHp52hvHwReFK51VUerBkhJ1SdF_0hYHks5qNlx-gaJpZM4Iwur8>
> .
>